knack/output.py

```diff
@@ -25,7 +25,7 @@ def _decode_str(output):
 class _ComplexEncoder(json.JSONEncoder):
 
     def default(self, o):  # pylint: disable=method-hidden
-        if isinstance(o, bytes) and not isinstance(o, str):
+        if (isinstance(o, bytes) or isinstance(o, bytearray)) and not isinstance(o, str):
```

Comment:
In Python 3, if `o` is `bytes` or `bytearray`, it can't be `str`, so `o` needs to be decoded.

In Python 2, `bytes` is just an alias for `str`, so `o.decode()` is skipped.

Since we have dropped Python 2, we can safely drop `not isinstance(o, str)`：

```suggestion
        if isinstance(o, bytes) or isinstance(o, bytearray):
```